Cannot find module 'src/types' or its corresponding type declarations

Minimal reproduction:

```
$ npm install typescript@4.7.3 @types/node@16.11.39 iter-tools-es@7.3.3 --no-save
$
$ echo "import { apply } from 'iter-tools-es';" > index.ts
$
$ npx tsc index --noEmit
node_modules/iter-tools-es/impls/apply/apply.d.ts:1:27 - error TS2307: Cannot find module 'src/types' or its corresponding type declarations.

1 import { Wrappable } from 'src/types';
                            ~~~~~~~~~~~


Found 1 error in node_modules/iter-tools-es/impls/apply/apply.d.ts
```

--- a/src/impls/apply/apply.d.ts
+++ b/src/impls/apply/apply.d.ts
@@ -1,4 +1,4 @@
-import { Wrappable } from 'src/types';
+import { Wrappable } from '../../types';
 
 declare function apply<Return, A>(fn: (...args: Array<A>) => Return, args: Wrappable<A>): Return;
 

